com.unity.render-pipelines.high-definition/Runtime/Compositor/ChromaKeying.cs

```diff
@@ -27,9 +27,7 @@ internal class ShaderIDs
 
         public override void Setup()
         {
-            var hdrpAsset = HDRenderPipeline.defaultAsset;
-            if (hdrpAsset != null)
-                m_Material = CoreUtils.CreateEngineMaterial(hdrpAsset.renderPipelineResources.shaders.chromaKeyingPS);
+            m_Material = CoreUtils.CreateEngineMaterial(HDRenderPipelineGlobalSettings.instance.renderPipelineResources.shaders.chromaKeyingPS);
```

Comment:
Before we checked against null asset. It is not done anymore. The HDRenderPipelineGlobalSettings.instance can still be null?